src/pages/steps/Steps.tsx

```diff
@@ -94,18 +94,20 @@ const StepWrapper = ({
   ...props
 }: StepWrapperProps) => {
   return (
-    <Stack spacing={4} {...getStackProps(props)}>
-      <Title
-        color={getValue('title.color')}
-        lineHeight="220%"
-        alignItems="center"
-        spacing={3}
-        css={`
-          border-bottom: 1px solid ${getValue('title.borderColor')};
-        `}
-      >
+    <Stack
+      paddingX={5}
+      paddingY={4.5}
+      borderRadius={getGlobalBorderRadius}
+      backgroundColor="white"
+      css={`
+        box-shadow: 0px 4px 16px 0px #00000012;
```

Comment:
Is this hex correct?

Reply:
Yes, but I used a theme value instead of a hardcoded value.